regulations/generator/section_url.py

```diff
@@ -34,11 +34,13 @@ def fetch(self, citation, version, sectional):
             url = ''
 
             if sectional:
-                view_name = 'chrome_section_view'
+                view_name = 'reader_view'
                 label = self.view_label_id(citation, version)
+                part = label.split("-")[0]
+                section = label.split("-")[1]
```

Comment:
It looks like `citation` is already an array of regulation identifiers and can be used in place of label.
```suggestion
                part = citation[0]
                section = citation[1]
```